### PR TITLE
Better logging in RibonucleotideDB.cpp

### PR DESCRIPTION
**Pull request by poshul**
Added DEBUG level output to show which Custom_RNA_modifications and Modomics files we are using. Fix level of output for unequal RNA bases masses from WARNING to DEBUG

## Checklist
- [ ] Make sure that you are listed in the AUTHORS file
- [ ] Add relevant changes and new features to the CHANGELOG file
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] New and existing unit tests pass locally with my changes
- [ ] Updated or added python bindings for changed or new classes (Tick if no updates were necessary.)

### How can I get additional information on failed tests during CI
<details>
  <summary>Click to expand</summary>
If your PR is failing you can check out

- The details of the action statuses at the end of the PR or the "Checks" tab.
- http://cdash.openms.de/index.php?project=OpenMS and look for your PR. Use the "Show filters" capability on the top right to search for your PR number.
  If you click in the column that lists the failed tests you will get detailed error messages.

</details>

### Advanced commands (admins / reviewer only)
<details>
  <summary>Click to expand</summary>
  
- `/reformat` (experimental) applies the clang-format style changes as additional commit. Note: your branch must have a different name (e.g., yourrepo:feature/XYZ) than the receiving branch (e.g., OpenMS:develop). Otherwise, reformat fails to push.
- setting the label "NoJenkins" will skip tests for this PR on jenkins (saves resources e.g., on edits that do not affect tests)
- commenting with `rebuild jenkins` will retrigger Jenkins-based CI builds
  
</details>

---
:warning: Note: Once you opened a PR try to minimize the number of *pushes* to it as every push will trigger CI (automated builds and test) and is rather heavy on our infrastructure (e.g., if several pushes per day are performed).

### CODE REVIEW

**Comment by timosachsenberg** [comment]
hi can you add some background why is this not warning anymore?

**Comment by poshul** [comment]
Sure. When I wrote the code I assumed that the modomics json file would
have the masses of all the modifications consistent with the structural
formula. Unfortunately this isn't true of a good number of the
modifications, where the masses were left blank, or simply were wrong.
Since this appears to be the "intended" behaviour from modomics side I've
changed the warning to a debug note, so that I can still look at them when
we update the modomics dB in the future.

On Monday, June 5, 2023, Timo Sachsenberg ***@***.***> wrote:
> hi can you add some background why is this not warning anymore?
>
> —
> Reply to this email directly, view it on GitHub, or unsubscribe.
> You are receiving this because you authored the thread.<
https://ci3.googleusercontent.com/proxy/MHRwYb3ome4b6Z5QQ_FE0XYfefOSd-974U4p0UQxX5S0p814oXQPpZw7r6BjO9KKY_K8k66QnIETScHHjEyixDD0sEVlbEWGN3zZTv25m9bJtQed9vxkq_PKFTYH_r_Riuj28drGiLm38MJ8I0_92nGlmPa9FwpNZ8z3hCVnVIOWVmSiRAxK6eYAlwb20OdgpF5mw25pdYC_ZNFo62nxsPWSJf_Y27sNOaPNyaBsVum5o9_iAg=s0-d-e1-ft#https://github.com/notifications/beacon/AABD3C76ELU7MGCSI3VSP63XJV7JZA5CNFSM6AAAAAAYUOEQG2WGG33NNVSW45C7OR4XAZNMJFZXG5LFINXW23LFNZ2KUY3PNVWWK3TUL5UWJTS56IS3M.gif>Message
ID: ***@***.***>

File: src/openms/source/CHEMISTRY/RibonucleotideDB.cpp [hunk 1]
```diff
@@ -57,8 +57,12 @@ namespace OpenMS
   {
     // Modomics mods were retreived from https://www.genesilico.pl/modomics/api/modifications
     readFromJSON_("CHEMISTRY/Modomics.json");
+    OPENMS_LOG_DEBUG << "Loading modomics RNA Modifications from "<<  File::find("CHEMISTRY/Modomics.json") <<"\n";
+    
     // We still use the old tsv format for custom mods
     readFromFile_("CHEMISTRY/Custom_RNA_modifications.tsv");
+    OPENMS_LOG_DEBUG << "Loading custom RNA Modifications from"<<  File::find("CHEMISTRY/Custom_RNA_modifications.tsv") <<"\n";
```
Comment (cbielow) [inline src/openms/source/CHEMISTRY/RibonucleotideDB.cpp:64]:
```suggestion
    OPENMS_LOG_DEBUG << "Loading custom RNA Modifications from "<< File::find("CHEMISTRY/Custom_RNA_modifications.tsv") <<"\n";
```

File: src/openms/source/CHEMISTRY/RibonucleotideDB.cpp [hunk 2]
```diff
@@ -57,8 +57,12 @@ namespace OpenMS
   {
     // Modomics mods were retreived from https://www.genesilico.pl/modomics/api/modifications
     readFromJSON_("CHEMISTRY/Modomics.json");
+    OPENMS_LOG_DEBUG << "Loading modomics RNA Modifications from "<<  File::find("CHEMISTRY/Modomics.json") <<"\n";
```
Comment (cbielow) [inline src/openms/source/CHEMISTRY/RibonucleotideDB.cpp:60]:
```suggestion
    OPENMS_LOG_DEBUG << "Loading modomics RNA Modifications from "<< File::find("CHEMISTRY/Modomics.json") <<"\n";
```

**Pull request merged**